amo2kinto/generator.py

```diff
@@ -35,7 +35,7 @@ def sort_records(record):
     filename = os.path.join(target_dir, 'index.html')
 
     logger.info('Writing %s' % filename)
-    with open(filename, 'w') as f:
+    with open(filename, 'wb') as f:
```

Comment:
`res` is a unicode string so it's correct to use `wb` if you're going to convert it to a byte string. But I would instead keep it as a unicode string. 

I.e.
```python
res = '🚵‍♂️'
with open('/tmp/index.html', 'w') as f:
    f.write(res)
```
This works.

Reply:
Why do you prefer this? It seems natural to me to write bytes to disk, since after all a disk contains a string of bytes, not a string of code points.